manifests/appcontroller.yaml

```diff
@@ -3,10 +3,10 @@ kind: Pod
 metadata:
   name: k8s-appcontroller
   annotations:
-    pod.alpha.kubernetes.io/init-containers: '[{"name": "kubeac-bootstrap", "image": "mirantis/k8s-appcontroller", "command": ["bootstrap", "/opt/kubeac/manifests"]}]'
+    pod.alpha.kubernetes.io/init-containers: '[{"name": "kubeac-bootstrap", "image": "mirantis/k8s-appcontroller:single-binary", "command": ["kubeac", "bootstrap", "/opt/kubeac/manifests"]}]'
```

Comment:
If we are to merge it to master, we should not point to tagged docker image but use trunk